modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,109 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+<% if standalone -%>
```

Comment:
Can we not just use code to determine if its standalone? 
Given we already shell out to find the secondaries the logic there could determine if its stand alone or not.